service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -157,6 +157,13 @@ public void setOffRampReason(Claim claimWithOffRamp) {
     Optional<ClaimSubmissionEntity> claimSubmission =
         claimSubmissionRepository.findFirstByReferenceIdAndIdTypeOrderByCreatedAtDesc(
             claimWithOffRamp.getCollectionId(), claimWithOffRamp.getIdType());
+    if (claimSubmission.isEmpty()) {
+      log.info(
```

Comment:
Before Yoom asks, yes this is info not error. There are cases we offramp and this doesn't exist yet. When i collapsed the MasIntegration routes for offramping I found that out.